test/setupTest.ts

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
Does this file need to be in the test root, or can we move it to the webui folder?